pkg/app/piped/executor/ecs/ecs.go

```diff
@@ -157,6 +184,42 @@ func applyServiceDefinition(ctx context.Context, cli provider.Client, serviceDef
 	return service, nil
 }
 
+func applyStandaloneTask(
+	ctx context.Context,
+	cli provider.Client,
+	clusterDefinition types.Cluster,
+	awsVpcConfiguration types.AwsVpcConfiguration,
+	taskDefinition types.TaskDefinition,
+) (*ecs.RunTaskOutput, error) {
```

Comment:
We don't use this `ecs.RunTaskOuput` anyway so I think it's better to just return the error.
ref: https://github.com/pipe-cd/pipecd/pull/4084/files#diff-0ced6421324def8505cb6dadaa49f469d0855ef22437ba6d8ad7f5279a61b2e4R215

Reply:
I think you are right. I'll fix it🚀